### PR TITLE
media-sound/ncmpcpp: add 0.9.2_p20221231, bump 9999 EAPI 7->8

### PR DESCRIPTION
**Pull request by NexAdn**
In response to #33808. Upstream hasn't published a real release for some time now.

### CODE REVIEW

File: media-sound/ncmpcpp/ncmpcpp-0.9.2_p20221231.ebuild [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools flag-o-matic
+
+NCMPCPP_COMMIT="9f44edf0b1d74da7cefbd498341d59bc52f6043f"
+
+DESCRIPTION="featureful ncurses based MPD client inspired by ncmpc"
```
Comment (mm1ke) [inline media-sound/ncmpcpp/ncmpcpp-0.9.2_p20221231.ebuild:10]: Capital first letter:
```suggestion
DESCRIPTION="Featureful ncurses based MPD client inspired by ncmpc"
```

File: media-sound/ncmpcpp/ncmpcpp-0.9.2_p20221231.ebuild [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	net-misc/curl
+	sys-libs/ncurses:=[unicode(+)]
+	sys-libs/readline:=
+	taglib? ( media-libs/taglib )
+	visualizer? ( sci-libs/fftw:3.0= )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+S="${WORKDIR}/${PN}-${NCMPCPP_COMMIT}"
```
Comment (mm1ke) [inline media-sound/ncmpcpp/ncmpcpp-0.9.2_p20221231.ebuild:32]: could you move `S` after `SRC_URI`

File: media-sound/ncmpcpp/ncmpcpp-0.9.2_p20221231.ebuild [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+src_configure() {
+	filter-lto
+
+	econf \
+		$(use_enable clock) \
+		$(use_enable outputs) \
+		$(use_enable visualizer) \
+		$(use_with lto) \
+		$(use_with taglib) \
+		$(use_with visualizer fftw)
```
Comment (mm1ke) [inline media-sound/ncmpcpp/ncmpcpp-0.9.2_p20221231.ebuild:51]:
```suggestion
	local myeconfargs=(
		$(use_enable clock)
		$(use_enable outputs)
		$(use_enable visualizer)
		$(use_with lto)
		$(use_with taglib)
		$(use_with visualizer fftw)
	)
	econf "${myeconfargs[@]}"
```

**Comment by spencermp** [comment]
Just a bump in hopes for fixes and/or merger

**Review by juippis — APPROVED** [review]
Yeah, please implement the suggestions on next bump.

**Pull request closed without merging**